plugins/inputs/disk/README.md

```diff
@@ -17,6 +17,9 @@ Note that `used_percent` is calculated by doing `used / (used + free)`, _not_
 
   ## Ignore mount points by filesystem type.
   ignore_fs = ["tmpfs", "devtmpfs", "devfs", "iso9660", "overlay", "aufs", "squashfs"]
+
+  ## Ignore mount points by mount options.
+  ignore_mount_opts = ["bind"]
```

Comment:
The default config options should match what is used and be commented out. I do not believe we should be removing any mount with the `bind` option by default and instead leave that to the user.

I have never seen a `bind` option on a mount and it appears that this is something that `gopsutil` [adds to the options](https://github.com/shirou/gopsutil/blob/cf6309316c89a96a0a4b848185b32830a6e8175e/disk/disk_linux.go#L323). It makes me a little wary of making this change if we ever go away from gopsutil, even if that is unlikely.

I think this is worth a comment in the docs as this is pretty non-standard:

```suggestion
  ## Ignore mount points by mount options.
  ## The `mount` command reports options of all mounts in parathesis.
  ## Bind mounts can be ignored with the special `bind` option.
  # ignore_mount_opts = []
```

Reply:
Makes sense. 

Good callout on being conservative with bind mounts. FWIW from the mount(8) manpage it seems that it's possible, but not common, to specify `bind` as a mount option, although mount(8) handles it on its own and doesn't pass it to the kernel directly. So I think it's a bit of a convention to include `bind` along with other mount options.